src/cloud-element-templates/ElementTemplatesPropertiesProvider.js

```diff
@@ -48,7 +53,7 @@ export default class ElementTemplatesPropertiesProvider {
       };
 
       // (1) Add templates group
-      addGroupsAfter('documentation', groups, [ templatesGroup ]);
+      addGroupsAfter(ALWAYS_DISPLAYED_GROUPS, groups, [ templatesGroup ]);
```

Comment:
As a side effect, the template group is displayed above "Documentation". There were no tests about this, so I assumed the value was included by accident. Also, in the old properties panel "Catalog" was displayed above "Documentation".

Reply:
Documentation is gone as soon as one applies a template.